operators/text/string_remove.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "string_remove.hpp"
+#include "string_tensor.h"
+#include <vector>
+#include <locale>
+#include <codecvt>
+#include <algorithm>
+
+
+KernelStringRemove::KernelStringRemove(OrtApi api, const OrtKernelInfo* /*info*/) : BaseKernel(api) {
+}
+
+void KernelStringRemove::Compute(OrtKernelContext* context) {
+  // Setup inputs
+  const OrtValue* input_string = ort_.KernelContext_GetInput(context, 0);
+  const OrtValue* input_condition = ort_.KernelContext_GetInput(context, 1);
+
+  OrtTensorDimensions string_dimensions(ort_, input_string);
+  OrtTensorDimensions condition_dimensions(ort_, input_condition);
+
+  if ((!(string_dimensions.IsScalar() || string_dimensions.IsVector())) ||
+      (!(condition_dimensions.IsVector() || condition_dimensions.IsVector()))) {
```

Comment:
Why is this `condition_dimensions.IsVector() || condition_dimensions.IsVector()` instead of `condition_dimensions.IsScalar() || condition_dimensions.IsVector()`?

Reply:
Fixed, thank you for pointing out this bug.